themes/startwords/layouts/partials/header.html

```diff
@@ -1,8 +1,16 @@
 <header>
     <nav aria-label="main">
         <ul>
-            <li><a class="issues" href="/issues">Issues</a></li>
-            <li><a class="home" href="/">Home</a></li>
+            <li class="issues">
```

Comment:
Would it make sense to configure the menu in the site config ? You can see the footer for an example

Reply:
we could do this, but there seem to be at least two reasons not to:
- theoretically this menu won't ever change (changing it would at least require new design) and such a change would be pretty major in terms of content architecture
- one of the two items needs to be "special" because it's an image with some accessible text rather than just text anyway

Reply:
Right - it wouldn't make sense for the logo to change, and it doesn't get us a lot to make the 'issues' part of the menu configurable.